tools/docker/all/Dockerfile

```diff
@@ -28,7 +28,6 @@ RUN apt-get update && apt-get install -y \
 	libnl-3-dev \
 	libnl-route-3-dev \
 	libnl-genl-3-dev \
-	libzmq3-dev \
```

Comment:
Note that there's still zmq references in:
- .devcontainer/Dockerfile
- TODO in framework/transport/ieee1905_transport/test/CMakeLists.txt

Also, cmake/Findzmq.cmake can be removed.

Reply:
Done